orangefs/tasks/main.yml

```diff
@@ -0,0 +1,87 @@
+---
```

Comment:
It might be tidier syntax to conditionally include task files for client, server, etc. after the common tasks here.